admin/release.py

```diff
@@ -617,6 +651,8 @@ def publish_artifacts_main(args, base_path, top_level):
     scratch_directory.child('rpm').createDirectory()
     scratch_directory.child('python').createDirectory()
     scratch_directory.child('pip').createDirectory()
+    scratch_directory.child('homebrew').createDirectory()
+    homebrew_repo_url = "git@github.com:ClusterHQ/homebrew-tap.git"
```

Comment:
Has this been tested? It looks like there isn't a way to run this against a test repository?